codalab/apps/web/models.py

```diff
@@ -258,6 +258,8 @@ class Competition(models.Model):
     require_team_approval = models.BooleanField(default=True, verbose_name="Organizers need to approve the new teams")
     teams = models.ManyToManyField(Team, related_name='competition_teams', blank=True, null=True)
 
+    competition_docker_image = models.CharField(max_length=128, default='', blank=True)
```

Comment:
`default=settings.DEFAULT_WORKER_IMAGE` or what-have-you

Reply:
This doesn't really make sense though in this case though I think? We check if the organizer has actually set something here rather than being blank. If we set it to the default we would always be comparing it to the setting-default to see if it was different later on in the code?

Reply:
that makes sense